src/Psalm/Internal/LanguageServer/Server/TextDocument.php

```diff
@@ -121,23 +121,16 @@ public function didChange(VersionedTextDocumentIdentifier $textDocument, array $
             return;
         }
 
-        if ($this->project_analyzer->onchange_line_limit === 0) {
-            return;
-        }
```

Comment:
didChange will never be called when `onchange_line_limit=0` because
https://github.com/vimeo/psalm/blob/e91e33c39c2d15f326f91da7906744714b387a65/src/Psalm/Internal/LanguageServer/LanguageServer.php#L269